src/scene/graphics/scene-grab.js

```diff
@@ -261,7 +261,12 @@ class SceneGrab {
                     }
 
                     // reallocate RT if needed
-                    if (self.shouldReallocate(this.depthRenderTarget, camera.renderTarget?.depthBuffer)) {
+                    if (
+                        self.shouldReallocate(
```

Comment:
I think a simpler fix might be to pass a sourceRenderTarget to shouldReallocate, instead of a source texture, and that way it will get the size in all cases. So in this case, just `camera.renderTarget` instead of `camera.renderTarget?.depthBuffer ?? camera.renderTarget?.colorBuffer`. All 4 calls to this function in the file would need to be adjusted the same simple way.

Reply:
Done